locales/en-US/content.json

```
@@ -82,6 +82,36 @@
     "askClear": "Which message do you want to clear?",
     "toClear": "The message to clear",
     "cleared": "Cleared {{value}}"
+  },
+  "role": {
+    "missingPerms": "You need the permission to manage the server in order to use this command.",
+    "everyone": "You cannot make @.everyone a level role.",
```

the `.` is unnecessary here because we can (and have) limited which mentions are parsed by Discord.
```typescript
allowedMentions: { parse: [] },
``` 
prohibits Discord from parsing the mention
```suggestion
    "everyone": "You cannot make @everyone a level role.",
```